build.sbt

```diff
@@ -456,7 +456,8 @@ lazy val resolverSettings = Seq(
   resolvers ++= Seq(
     "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/",
     "Sonatype snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
-    "Sonatype releases" at "https://oss.sonatype.org/content/repositories/releases"
+    "Sonatype releases" at "https://oss.sonatype.org/content/repositories/releases",
+    "Scala PR" at "https://scala-ci.typesafe.com/artifactory/scala-pr-validation-snapshots"
```

Comment:
it might be better not to merge this, since the more resolvers you have, the longer `update` takes. otherwise LGTM

Reply:
True. Removed.